### DISCUSSION TITLE
NTSC-RT Ultimate Preset Pack v0.2.0

### DISCUSSION THREAD

**Discussion by polikux** [ntsc-rs/ntsc-rs#207] [General]
![NTSC](https://github.com/user-attachments/assets/897b6b3e-397e-4571-b650-f28bfc5c0d1f)
[Download](https://download1072.mediafire.com/kv6d1lbp98kgEI80Pm-rrJCAQgYa5VRyrCJDqMP_Iz9F4_HUu6bvwTnrBNczLjyh7h0QGqW1wij8rahF1JdapvnFZR0xQJuYc0cSZY1vduc2azlWMS5vdl2NyXcgM8AQTgYksGu7HeQ4vqFr8i0ttUS8t4DHBy4esRV8tZEw-Q/ocv4jicddot53fu/NTSC-RT+Ultimate+Preset+Pack+V0.2.0.zip) NTSC-RT Ultimate Preset Pack v0.2.0 📼💾

> [!NOTE]
> I had to put this file on Mediafire because it is over 25 MB. There are no viruses there.

# Introduction

Howdy. Here is version **0.2.0** of **NTSC-RT Ultimate Preset Pack**. I know I haven't updated this project in a long time, I hope you will enjoy this one of the biggest updates of this project. I have plans to set up my own repository for this preset pack but we'll see if that happens.

I warn you that there may be some mistakes because quite a lot has been added. If I forgot to add or do something, write to me.

I added some important things like:

----

### Preset renders
![TestimagePR](https://github.com/user-attachments/assets/7d3be684-f433-4a13-887d-3e2fc9173215)
Since this update, in each preset author folder of a given category there is a folder called "Renders" which contains a preview of all effects on the rendered image. The photo could probably be more, if you can call it that, "Test" like a test card, but it will certainly be enough to check how a given effect works.

> [!TIP]
> Feel free to use this photo to test your presets! ❤

<details>
<summary>Test photo without effects</summary>

![TESTIMAGE](https://github.com/user-attachments/assets/68a15ead-d1f6-4061-81de-62b09e14fc59)

</details>

----

### Folder (All Presets)
This folder contains all the effects included in this pack. It is there to put all of its contents (Except "0Explanation") into the presets folder in NTSC-RS for easier use of the prepared presets.

> [!NOTE]
> To properly navigate through the presets in this folder, learn these shortcuts. They are located at the beginning of each preset and indicate its categories.
>
>- AD' = Analog Drone Effects
>- TV' = Analog TV
>- CAM' = Camcorders VHS-C
>- CC' = Capture Card
>- RCA' = RCA Connectors
>- TAPE' =Tapes

----
### The rest of the changes
- Removed "Damaged Tapes" folder
- Updated some presets and added new ones.
- For a full list, please see "CHANGELOG"

**Comment by polikux** [comment]
> I forgot to change the ascii code image in REDMI version name. Don't be offended.

**Comment by notdigidani** [comment]
this is so pretty oh my goooood

**Comment by smfthe1199** [comment]
Thanks for updating this.

**Comment by Pleiades-Spl** [comment]
Could you make this a repository instead of making a new discussion every time there is a new release?

**Comment by guest10074** [comment]
why mediafire, their downloads speeds are sometimes godawful
